GitHub_Learning/Feature/Onboarding/Presentation/OnboardingPresenter.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  OnboardingPresenter.swift
+//  GitHub_Learning
+//
+//  Created by Amin Ashhab on 09/11/2020.
+//  Copyright © 2020 Amin_Second_Test_Project. All rights reserved.
+//
+
+import Foundation
+import CleanCore
+import CleanPlatform
+
+protocol OnboardingPresenter: Presenter {
+    
+}
+
+final class OnboardingPresenterImpl: BasePresenter<OnboardingView> {
+    
+    private let onboardingController: OnboardingController
+    
+    init(
+        onboardingController: OnboardingController
+    ) {
+        self.onboardingController = onboardingController
+    }
+    
+    func viewDidLoad() {
+        
```

Comment:
Tady by to chtelo volat `onboardingController.loadPages()` a nastavovat je na view